packages/flutter/lib/src/cupertino/text_field.dart

```diff
@@ -821,6 +837,7 @@ class CupertinoTextField extends StatefulWidget {
     properties.add(DiagnosticsProperty<Clip>('clipBehavior', clipBehavior, defaultValue: Clip.hardEdge));
     properties.add(DiagnosticsProperty<bool>('scribbleEnabled', scribbleEnabled, defaultValue: true));
     properties.add(DiagnosticsProperty<bool>('enableIMEPersonalizedLearning', enableIMEPersonalizedLearning, defaultValue: true));
+    properties.add(DiagnosticsProperty<SpellCheckConfiguration>('spellCheckConfiguration', spellCheckConfiguration, defaultValue: SpellCheckConfiguration.disabled));
```

Comment:
Should the defaultValue always match the default value in the constructor? Maybe make `SpellCheckConfiguration.disabled` into a const variable like `defaultSpellCheckConfiguration`.